### PR TITLE
Add a message to the Lobby command

### PR DESCRIPTION
**Pull request by Lasergott**

### CODE REVIEW

File: TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/commands/LobbyCommand.java [hunk 1]
```diff
@@ -21,6 +21,8 @@ public void execute(CommandSender sender, String[] args) {
         }
         ProxiedPlayer player = (ProxiedPlayer) sender;
         ServerInfo serverInfo = TimoCloudBungee.getInstance().getLobbyManager().searchFreeLobby(player.getUniqueId(), player.getServer().getInfo());
-        if (serverInfo != null) player.connect(serverInfo);
+        if (serverInfo != null)
+            player.connect(serverInfo);
+            BungeeMessageManager.sendMessage(sender,TimoCloudBungee.getInstance().getFileManager().getConfig().getString("LobbyCommandMessage"));
```
Comment (TimoZikeli) [inline TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/commands/LobbyCommand.java]: One-line if syntax does not work with two lines - use brackets

File: TimoCloud-Universal/src/main/resources/bungeecord/config.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ fallbackGroup: 'Lobby'
 useFallback: true
 emergencyFallback: 'Economy'
 LobbyChooseStrategy: "RANDOM"
+LobbyCommandMessage: 'Successful connected to a Lobby server'
```
Comment (TimoZikeli) [inline TimoCloud-Universal/src/main/resources/bungeecord/config.yml]: We do not want messages in `config.yml` - create an extra messages file

**Review by TimoZikeli — CHANGES_REQUESTED** [review]

File: TimoCloud-Universal/src/main/resources/bungeecord/config.yml (hunk 2 above)
Comment (TimoZikeli) [reply TimoCloud-Universal/src/main/resources/bungeecord/config.yml]: Please use proper english - "Successfully" ;)

File: TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/commands/LobbyCommand.java (hunk 1 above)
Comment (TimoZikeli) [reply TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/commands/LobbyCommand.java]: We also want messages if there is no free server or if the player already is on the only free Lobby.

File: TimoCloud-Universal/src/main/resources/bungeecord/messages.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+LobbyConnect: '&aSuccessfully connected to a Lobby server'
```
Comment (TimoZikeli) [inline TimoCloud-Universal/src/main/resources/bungeecord/messages.yml]: ‚&aSuccessfully connected to &6{server_name}‘

File: TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/commands/LobbyCommand.java [hunk 4]
```diff
@@ -21,6 +22,11 @@ public void execute(CommandSender sender, String[] args) {
         }
         ProxiedPlayer player = (ProxiedPlayer) sender;
         ServerInfo serverInfo = TimoCloudBungee.getInstance().getLobbyManager().searchFreeLobby(player.getUniqueId(), player.getServer().getInfo());
-        if (serverInfo != null) player.connect(serverInfo);
+        if (serverInfo == null) {
+            BungeeMessageManager.sendMessage(sender, TimoCloudBungee.getInstance().getFileManager().getMessages().getString("NoFreeLobbyFound"));
+            return;
+        }
+        player.connect(serverInfo);
+        BungeeMessageManager.sendMessage(sender, TimoCloudBungee.getInstance().getFileManager().getConfig().getString("LobbyConnect"));
```
Comment (TimoZikeli) [inline TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/commands/LobbyCommand.java]: .replace(„{server_name}“, serverInfo.getName());

**Review by TimoZikeli — CHANGES_REQUESTED** [review]

**Comment by Lasergott** [comment]
It is okay now? xD

**Review by TimoZikeli — APPROVED** [review]

**Pull request merged**